cypress/objects/timelog.js

```diff
@@ -6,7 +6,23 @@ cy.url().should('include', '/time-logging');
 pagetitle(){
 return cy.get('.page__title')
 }
-
+calendarv(){
+return cy.get('.calendar')
+}
+getTile(){
```

Comment:
What tile is this? If this is the time entry form then please update the name of the function to reflect that :) 

Something like `getTimeEntryForm`